EMS/core-bundle/src/Controller/Api/Search/SearchController.php

```diff
@@ -103,14 +103,48 @@ public function getIndicesFromAlias(Request $request): Response
         $json = Json::decode((string) $request->getContent());
         $alias = $json['alias'] ?? null;
         if (!\is_string($alias)) {
-            throw new \RuntimeException('Unexpected: index must be a string');
+            throw new \RuntimeException('Unexpected: alias must be a string');
         }
 
         return new JsonResponse([
             'indices' => $this->elasticaService->getIndicesFromAlias($alias),
         ]);
     }
 
+    public function getIndicesForContentTypes(Request $request): Response
+    {
+        $json = Json::decode((string) $request->getContent());
+        $aliases = $json['aliases'] ?? null;
+        if (!\is_array($aliases)) {
+            throw new \RuntimeException('Unexpected: aliases must be an array');
+        }
+
+        return new JsonResponse([
+            'indices' => $this->elasticaService->getIndicesForContentTypes($aliases),
+        ]);
+    }
+
+    public function filterStopWords(Request $request): Response
+    {
+        $json = Json::decode((string) $request->getContent());
+        $index = $json['index'] ?? null;
+        $analyzer = $json['analyzer'] ?? null;
+        $words = $json['words'] ?? null;
+        if (!\is_string($index)) {
+            throw new \RuntimeException('Unexpected: index must be a string');
+        }
+        if (!\is_string($analyzer)) {
+            throw new \RuntimeException('Unexpected: analyzer must be a string');
+        }
+        if (!\is_array($words)) {
+            throw new \RuntimeException('Unexpected: words must be an array');
+        }
```

Comment:
Use https://github.com/ems-project/elasticms/blob/5.x/EMS/helpers/src/Standard/Type.php

Maybe we can add an 'array' method on the Type class

```
        $index = Type::string($json['index']);
        $analyzer = Type::string($json['analyzer']);
```

Reply:
Great!